monkey/monkey_island/cc/services/authentication_service/flask_resources/register_resources.py

```diff
@@ -17,7 +17,9 @@ def register_resources(api: flask_restful.Api, authentication_facade: Authentica
     )
     api.add_resource(Login, *Login.urls, resource_class_args=(authentication_facade,))
     api.add_resource(Logout, *Logout.urls, resource_class_args=(authentication_facade,))
-    api.add_resource(AgentOTP, *AgentOTP.urls)
+    # TODO: Get OTPGenerator after #3187 and #3190
+    otp_generator = None
```

Comment:
Maybe create and pass in a stub that returns a constant value so that the endpoint isn't constantly returning 500 erros.